authors/apps/profiles/urls.py

```diff
@@ -1,14 +1,19 @@
 from django.urls import path
+
 from .views import (
     UserProfileView,
     UserProfileUpdateView,
     UserProfileListView,
+    FollowsUpdateAPIView,
+    MyFollowersListView,
+    IFollowListView,
 )
 
-
 urlpatterns = [
     path('<username>', UserProfileView.as_view(), name='profile-detail'),
     path('<username>/edit', UserProfileUpdateView.as_view(), name='profile-update'),
     path('', UserProfileListView.as_view(), name='profile-list'),
-
+    path('Ifollow/', IFollowListView.as_view(), name='get_i_follow'),
```

Comment:
@ManuelDominic, this endpoint conflicts with `<username>`.
whenever there's a variable involved, ensure not to use that same path for any other endpoint